deploy.py

```diff
@@ -161,10 +154,12 @@ def deploy(ctx, target, mode, version=None):
     print(f"Starting AWS session for: {mode}")
     session = boto3.Session(profile_name=ctx.config.aws.profile_name)
     ecs_client = session.client('ecs', region_name='eu-central-1')
+    task_role_arn = ctx.config.aws.task_role_arn if not target == "harvester" else \
```

Comment:
Deze conditie leest niet zo lekker. Ik zou niet `if not - else` gebruiken, maar `if - else`.

En ik snap dat het nu nog kan, maar die clearlogins er ook nog bij en die moet dus ook conditioneel gaan worden. Gaan we dan niet in de loop van de tijd te veel verschillen krijgen tussen de twee targets? Kunnen we het updaten van het cluster niet beter extracten naar een functie en twee aparte deploy scripts maken. Dat lijkt mij overzichtelijker.

Reply:
Eens qua readability. Heb het veranderd. Ik zou voor nu even 1 if statement willen toevoegen met het mergen van clearlogins (wil ik best doen). Als we gaan refactoren zou ik prd, acc, dev tags willen meenemen en nadenken over hoe Github aan CD kan gaan doen. Tot die tijd denk ik dat de commands nog te managen zijn. Ik verwacht geen verdere toevoegingen. Als dat toch nog komt, kunnen we het even pokeren.

Reply:
Nu is het nog steeds `if not`, maar andere syntax. Ik bedoelde `if target == "service` of `if role == "harvester"` en dan die role arn's omdraaien 😉 .

En als er voorlopig geen verdere logica bij het deployen komt mag je het van mij zo houden. Ook niet teveel tijd aan besteden.